src/cart/entities/cart.entity.ts

```diff
@@ -34,28 +36,27 @@ export class Cart {
     default: () => 'CURRENT_TIMESTAMP',
   })
   @ApiProperty({ 
-    name: "updated_at",
+    name: "updatedAt",
     type: "string" ,
     format: "date-time",
     readOnly:true
   })
-  updated_at: Date;
+  updatedAt: Date;
 
   @BeforeUpdate()
   updateTimeStamp(){
-      this.updated_at = new Date;
+      this.updatedAt = new Date;
   }
 
-  @ManyToOne(type => User, user => user.id) user: User; 
-  @Column({name: 'user_id', type: "bigint"})
-  @JoinColumn({ name: 'user_id' })
+  @ManyToOne(type => User)
+  @JoinColumn({name: "user_id"})
   @ApiProperty({ 
     name: "userId",
     type: "integer",
     example: 1,
     description: "Id of the user the Cart belongs to"
   })
-  userId: number;
+  userId: User;
```

Comment:
Sebas, una pregunta, acá por qué el userId es de tipo User y no como estaba antes tipo number? como sale en el ejemplo de la documentación en @ApiProperty :thinking:

Reply:
![image](https://user-images.githubusercontent.com/38963060/145696114-aeb9358d-856f-47d1-b212-8aa279b93c74.png)
Es como se hace segun la documentacion oficial, especialmente cuando es un manyToOne - oneToMany.
![image](https://user-images.githubusercontent.com/38963060/145696164-f6411789-f059-4f96-ae21-d42d61c43315.png)

src: https://typeorm.io/#/many-to-one-one-to-many-relations